pyproject.toml

```diff
@@ -11,6 +11,8 @@ packages = [
 python = ">=3.8,<3.11"
 django = ">=3,<5"
 gcp-pilot = {version = "*", extras = ["tasks", "pubsub"]}
+factory-boy = "^3.2.1"
+freezegun = "^1.1.0"
```

Comment:
Since this is a library that will be largely used, it's dependencies should be **as loose as possible**, otherwise the applications will be incompatible very easily.

For instance:
If my django application requires freezegun==1.0, I cannot install this lib.